CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.12-dev2
+
+* Add `parser` parameter to `partition_html`.
+
 ## 0.4.12-dev1
```

Comment:
Could you change this line to `## 0.4.12-dev2` like you have above and add your bullet below?

Reply:
Done